@navikt/core/react/src/chat/Chat.tsx

```diff
@@ -1,7 +1,10 @@
 import React, { forwardRef, HTMLAttributes } from "react";
 import cl from "clsx";
 import Bubble, { BubbleProps } from "./Bubble";
-import { BodyLong, BodyShort } from "../typography";
+import { BodyLong } from "../typography";
+
+export const POSITIONS = ["left", "right"] as const;
+export const SIZES = ["medium", "small"] as const;
```

Comment:
Når vi bruker `react-docgen-typescript` for å generere prop-dokumentasjon hender det ofte at litt mer avanserte typer som her med `(typeof POSITIONS)[number]` ikke fungerer. 
Ser ut som docgen feiler ved sjekk av `Chat.tsx` (`yarn docgen`)

Selv om dette ofte bryter DRY-prinsippet har vi brukt `union`-types for det meste og ikke enum/enum-like løsninger for å unngå dette.

`react-docgen-typescript` er heller ikke så smart så eks dette: 
```ts
type TESTSIZE = "medium" | "small";
interface ChatProps{
  size?: TESTSIZE;
}
```
vil gi output 
```ts
props: {
	"size": { type: { size: "TESTSIZE" } }
}
```
når vi egentlig ønsker 
```ts
props: {
	"size": { type: { size: "\"medium\" | \"small\"" } }
}
```
for bedre dokumentasjon på Aksel.nav.no

Reply:
Skjønner.
Trodde egentlig Storybook var i stand til å generere options dynamisk basert på TS-typen, men ser ikke slik ut.